utils/download_data_filtered/downloadPlaces.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const child_process = require('child_process');
+const path = require('path');
+const fs = require('fs-extra');
+const async = require('async');
+const logger = require('pelias-logger').get('download_data_filtered');
+const parallelStream = require('pelias-parallel-stream');
+const streamArray = require('stream-array');
+const os = require('os');
+
+const maxInFlight = os.cpus().length * 10;
```

Comment:
This value is really high. Many/most developer machines these days are quad core with hyperthreading, so `os.cpus().length` will return 8, so there would be up to 80 downloads happening at once. Maybe up to 4 concurrent downloads can help ensure that network bandwidth is used effectively, but 80 is just overkill. The number of CPUs isn't really important either, so I'd suggest hardcoding this to between 2 and 4. Things should happen just as fast if not faster, and with much less thrashing of processes on users machine (mine is almost unusable when a download is happening).